Setup.hs

```diff
@@ -1,2 +0,0 @@
-import           Distribution.Simple
-main = defaultMain
```

Comment:
Is this removed because it isn't needed?

Reply:
Yes, it's not used when using `cabal-install`